includes/internal/emails/class-email-list-table.php

```diff
@@ -89,6 +89,9 @@ public function prepare_items( string $type = null ) {
 			'post_type'      => Email_Post_Type::POST_TYPE,
 			'posts_per_page' => $per_page,
 			'offset'         => $offset,
+			'meta_key'       => 'sensei_email_description', // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_key -- Query limited by pagination.
```

Comment:
I noticed that we have two different approaches in this feature too.
In my current task, as I need to use all those keys, I was going to unify them, prepending keys with "_" (underscore).

Reply:
For our case, I think this'll be the right approach. Probably the way to think is, if you expect the user to change it, no '_', otherwise, there should be one